### ISSUE TITLE
Question about training dataset

### ISSUE THREAD

**Issue by haodongyu** [pnavarre/eSR#2]
Thanks for the wonderful paper. I'd like to ask the follow questions to get better understand about your algorithm.
1. Do we need the validation set during training? if yes, where does it come from?
2. Does all the training and measurement process use the image in grayscale?

Thank you so much in advance.

**Comment by pnavarre** [comment]
Thanks for your feedback and sorry for the delay!
1. Technically, you do not need validation set during training because the setting we used (same as FSRCNN and ESPCN) runs a fixed number of iterations. Now, it doesn't hurt to use a validation set of your preference to identify problems and learning behaviors.
2. Yes, all training and inference run in grayscale according to the BT.609 standard. This is common in fast upscalers to reduce the number of computations.
